lib/build.py

```diff
@@ -199,12 +208,16 @@ def build_arg_get(arg):
    # Print summary & we’re done.
    if (ml.instruction_total_ct == 0):
       ch.FATAL("no instructions found: %s" % cli.file)
-   assert (ml.inst_prev.image_i + 1 == image_ct)  # should’ve errored already
+   if (cli.target is None):
+      assert ml.inst_prev.image_i + 1 == image_ct # should’ve errored already
+   elif (last_image is None):
+      ch.WARNING("--target: %s: not found; ignored" % cli.target)
```

Comment:
I feel like the target being not found should be an error? And you can probably figure out whether that's the case before building anything.